### PR TITLE
update  filesystem  tests

### PR DESCRIPTION
**Pull request by ftjahjad**
updating filesystems tests to restraint metadata
* cifs connectathon
* pjf-fstest
* loopdev
* nfs connectathon
* xfs

### CODE REVIEW

**Comment by ftjahjad** [comment]
@cki-bot  , please test

**Comment by ftjahjad** [comment]
@cki-bot , please test again

File: filesystems/cifs/connectathon/metadata [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+name=filesystems/cifs/connectathon
+owner=Jianhong Yin <jiyin@redhat.com>
+description=Connectathon for CIFS
+license=GPLv2
+confidential=no
+destructive=no
+
+[restraint]
+entry_point=bash ./runtest.sh
+dependencies=wget;git;samba;time;patch;cups;samba-client;cifs-utils;gcc;bzip2;nfs-utils;net-tools;rusers
```
Comment (emanchado) [inline filesystems/cifs/connectathon/metadata:11]: What happens with dependencies like `python3-lxml`?

**Review by emanchado — APPROVED** [review]
LGTM, but see comment.

File: filesystems/cifs/connectathon/metadata (hunk 1 above)
Comment (ftjahjad) [reply filesystems/cifs/connectathon/metadata]: Fixing another dependencies for filesystem tests.  :-)

**Comment by ftjahjad** [comment]
@cki-bot , please test again

**Comment by ftjahjad** [comment]
@cki-bot , please test again

**Comment by ftjahjad** [comment]
@rasibley , could you take a look,

**Comment by rasibley** [comment]
Hi @ftjahjad the bot wasn't pleased with xfstests, can you check it out ? :-)

**Comment by mh21** [comment]
The project is being moved to gitlab.com/cki-project/kernel-tests. Please reopen your merge request there :-)

**Pull request closed without merging**